pkg/referrerstore/oras/oras.go

```diff
@@ -117,30 +126,33 @@ func (store *orasStore) GetConfig() *config.StoreConfig {
 	return &store.rawConfig
 }
 
-func (store *orasStore) ListReferrers(ctx context.Context, subjectReference common.Reference, artifactTypes []string, nextToken string) (referrerstore.ListReferrersResult, error) {
-	// TODO: handle nextToken
-	registryClient, err := store.createRegistryClient(ctx, subjectReference)
+func (store *orasStore) ListReferrers(ctx context.Context, subjectReference common.Reference, artifactTypes []string, nextToken string, subjectDesc ...*ocispecs.SubjectDescriptor) (referrerstore.ListReferrersResult, error) {
```

Comment:
Is artifactTypes used in this method?

Reply:
It's currently not used because I believe v2 doesn't support artifact filtering yet. I was hesitant to remove it from the interface because we might want to add the functionality back once v2 has it built in? I know filtering is on the ORAS roadmap.